program-workflows.md

```diff
@@ -0,0 +1,6 @@
+## Program Workflows 
+This section provides more information on program specific workflows that are supported within UgandaEMR, and how to use the available forms, reports, dashboards and other utilties to achieve desired outcomes 
+
+The supported programs include:
+1. [DSDM](/differentiated-service-delivery-models-dsdm.md) 
+2. [HIV Recency Testing](/hiv-recency-testing.md)
```

Comment:
@ODORA0 delete this file which is a duplicate of the README.md

Reply:
@ssmusoke Done